packages/eslint-config-react-app/package.json

```diff
@@ -15,17 +15,20 @@
     "index.js"
   ],
   "peerDependencies": {
-    "@typescript-eslint/eslint-plugin": "2.x",
-    "@typescript-eslint/parser": "2.x",
+    "@typescript-eslint/eslint-plugin": "3.x",
```

Comment:
Maybe we should make these peers more specific to the versions that included eslint 7? i.e. 

`"eslint-plugin-jsx-a11y": "^6.3.0"`